src/Microsoft.Diagnostics.EventFlow.Inputs.EventSource/EventDataExtensions.cs

```diff
@@ -29,6 +30,11 @@ public static EventData ToEventData(this EventWrittenEventArgs eventSourceEvent)
             eventPayload["EventId"] = eventSourceEvent.EventId;
             eventPayload["EventName"] = eventSourceEvent.EventName;
             eventPayload["ActivityID"] = ActivityPathString(eventSourceEvent.ActivityId);
+            var eventContext = ContextResolver.GetRequestContext<object>();
+            if (eventContext != null)
+            {
+                eventPayload["EventContext"] = eventContext;
```

Comment:
1. It's not necessary request context. It's any context related to this Trace, so "Request" might be misleading
2. Since Trace API does not allow to pass it, I believe we cannot achieve it with trace input.  Could you elaborate?